testdata/fixtures/team/create_request.json

```diff
@@ -11,7 +11,6 @@
           "contacts":[
               {
                   "type":"slack",
-                  "displayName":"",
```

Comment:
i think doing this request to our API would actually result in a display name of "".  We should ensure this change is not having us lose the ability have an empty string display name as well as during `update` we can wipe out the current display name.